src/template_parser.h

```diff
@@ -465,8 +492,54 @@ class TemplateParser : public LexerHelper
                     (*m_template)[startOffset] == '-')
                 ++ startOffset;
         }
-        m_currentBlockInfo.range.startOffset = startOffset;
+
         m_currentBlockInfo.type = blockType;
+
+        if(blockType==TextBlockType::RawBlock)
+            startOffset = StripBlockRight(m_currentBlockInfo, matchStart);
+
+        m_currentBlockInfo.range.startOffset = startOffset;
+    }
+
+
+    size_t StripBlockRight(TextBlockInfo& currentBlockInfo, size_t position)
+    {
+        bool doTrim = m_settings.trimBlocks && (m_currentBlockInfo.type == TextBlockType::Statement || m_currentBlockInfo.type == TextBlockType::RawBlock );
```

Comment:
> TextBlockType::RawBlock )
Please, use the clang-format with the provided format file.

Reply:
When I use that format, the file has a lot of changes, is that ok?

There are changes in the include order, the position of the curly brackets, `++ var` changes to `++var`, and the indentation inside the switches.

Reply:
Ok. I see. I'll do it by myself.